segment_sc_contrast-agnostic/run_inference_single_image.py

```diff
@@ -0,0 +1,352 @@
+"""
```

Comment:
Are there specific differences between this script and https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/blob/4694169352fb962f607e7684fe778677adc123dc/monai/run_inference_single_image.py to justify duplication?

Reply:
Scripts are the same. There is a note about that at the end of this [README](https://github.com/ivadomed/canproco/blob/114ad2cb3c77902999a3858c32671cd2565cc961/segment_sc_contrast-agnostic/README.md). I duplicated the script because I did not want to create dependency on another [repo's branch](https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/60#issuecomment-1753033082). I am aware that this duplication is suboptimal. We could move the [run_inference_single_image.py](https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/blob/4694169352fb962f607e7684fe778677adc123dc/monai/run_inference_single_image.py) script to the [ivadomed/utilities](https://github.com/ivadomed/utilities) repo to avoid further duplications. Tagging @naga-karthik.

Reply:
i think the plan is to have it inside SCT, right @naga-karthik? The faster we get it inside SCT, the faster we can remove the duplicates and all work on the SCT-based implementation

Reply:
Yes, the plan is to get it inside SCT. That is my action task for this week!